src/main/java/frc/trigon/robot/subsystems/shooter/ShooterConstants.java

```diff
@@ -0,0 +1,15 @@
+package frc.trigon.robot.subsystems.shooter;
+
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonFX;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+
+public class ShooterConstants {
+    private static final int masterMotorID = 3;
+    private static final int motor2ID = 4;
```

Comment:
change name as stated above

Reply:
Lowercase D.
Use three motor objects: left, right, and master; then set master to either left or right. 
See the climber for an example.

Reply:
yes. but the master is right. there is no choice on the matter

Reply:
Master is left:)